src/index.css

```diff
@@ -49,6 +49,10 @@ a {
   cursor: pointer;
 }
 
+.cursor-pointer {
+  cursor: pointer;

```

Comment:
do not repeat same css ,instead of you can make utility class and use those classes